### DISCUSSION TITLE
Website not deploying on the custom domain!

### DISCUSSION THREAD

**Discussion by vishkpanicker** [vercel/vercel#6225] [Help]
Website is deploying on vercel.app domain.

Custom domain verified with nameservers.
Got valid configuration tick. 

Unfortunately, getting '404' from the custom domain

**Comment by fu-sen** [comment]
Are you looking at this display then?
![スクリーンショット 2021-05-13 22 47 46](https://user-images.githubusercontent.com/5434159/118135130-9f16ac00-b43d-11eb-812a-a7d05b2e0710.jpg)

This is the correct Not Found display returned by ▲Vercel.
If not, you are still looking at other servers, so check your domain name server settings carefully!

Or you may now look fine in your custom domain. The name server has a cache (TTL).

**Reply by vishkpanicker** [reply]
The domain is configured correctly with the nameservers. The SSL certificate is not getting generated. I tried with another custom domain, that's working fine. But the domain I'm trying to serve is not working.
![Screenshot 2021](https://user-images.githubusercontent.com/37177176/118139614-d2d6df00-b407-11eb-8f10-3e3ee45e9b32.png)

**Reply by fu-sen** [reply]
If your domain goes through Cloudflare, you need to set up a bit more:
https://vercel.com/support/articles/using-cloudflare-with-vercel

▲Vercel issues a Let's Encrypt certificate, but it is possible that you have disabled the CAA record settings.
Especially if your domain uses other web services or hosting and you move to ▲Vercel.

I think the others will be published over time. (But some time has passed)
If that doesn't work, we can investigate it specifically by publishing the domain.

**Reply by fu-sen** [reply]
If you set up your domain on May 10th, your domain may have been affected by this:
https://status.io/pages/maintenance/55957a99e800baa4470002da/609084a33c0c4a053c2da336
https://www.vercel-status.com/

**Reply by vishkpanicker** [reply]
I contacted Vercel support they informed me it was a bug from their side, and they resolved the issue. Thanks anyway :)
